core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/sqlite/SqliteDatabaseConnectionService.java

```diff
@@ -34,4 +36,9 @@ public String getDriver(SqliteDatabaseConnection databaseConnection) {
     public Class<SqliteDatabaseConnection> appliesTo() {
         return SqliteDatabaseConnection.class;
     }
+
+    @Override
+    public String refactorLimitAndOffset(SqliteDatabaseConnection databaseConnection, String query) {
+        return query;
+    }
```

Comment:
All XDatabaseConnectionService's extend either DatabaseConnectionService or SchemaDatabaseConnectionService (which extends DatabaseConnectionService). You could implement the default behavior in the DatabaseConnectionService class